app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -42,6 +43,11 @@ protected void onCreate(Bundle savedInstanceState) {
             setupDrawerContent(navigationView);
         }
 
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            getWindow().setStatusBarColor(getResources().getColor(R.color.color_primary));
```

Comment:
Yeah I was thinking about this. We would not be able to change the colour of every component by obtaining colors from api. Maybe yeah the primary colors but I don't know. @championswimmer you said you have worked on something like this before. Can you guide here?